Dockerfile

```diff
@@ -36,6 +34,23 @@ COPY --from=voila_rootfs_builder /etc/ /etc/
 
 
 FROM base_image_python AS kernel_rootfs_builder
+RUN apt-get install -y --no-install-recommends wget
```

Comment:
Should either move `wget` to `base_image_python`, or add an `apt-get update &&` in front.

Otherwise, what ends up happening is: Ubuntu upgrades package X from 1.0.1 to 1.0.2, and eventually deletes 1.0.1. `apt-get install` only relies on the local cache to get the latest version, and tries to download 1.0.1 that no longer exists.

Reply:
I'll do `apt-get` b/c Sergey doesn't want any of this in base_image_python.